.github/workflows/release.yml

```diff
@@ -41,11 +41,23 @@ jobs:
           fetch-depth: 0
           token: ${{ steps.app-token.outputs.token }}
 
-      - name: Setup Platform 🛠️
+      - name: Setup Bun 🛠️
         uses: oven-sh/setup-bun@4bc047ad259df6fc24a6c9b0f9a0cb08cf17fbe5 # v2
         with:
           bun-version: 1.1.18
 
+      - name: Setup Bun 🛠️
```

Comment:
that might be a mistake that we setup bun two times, right?

Reply:
This is for reliability

Reply:
Did you mean to respond to the other comment?